Add files via upload

JuanCarlosRamirez-dev opened the pull request.

JuanCarlosRamirez-dev committed:
Add files via upload

cyberhavok commented on Challenge_23/JuanCarlos_Ramirez_Osorio/clases-objeto.html:
```diff
@@ -0,0 +1,84 @@
+      a319.aceleracion = 10;
+      a319.desaceleracion = 5;
+
+      //Acelero 3 veces
+      a319.acelerar();
+      a319.acelerar();
+      a319.acelerar();
+      //Solicito un estado de velocidad
+      console.log(a319.estadoVelocidad());
+      //freno 1 vez
+      a319.frenar();
+      //Solicito un estado de velocidad
+      console.log(a319.estadoVelocidad());
+
+      a340 = new Aviones();
+      //Creo las propiedades
+      a340.nombre = "A340";
+      a340.aceleracion = 20;
+      a340.desaceleracion = 10;
+
+      //Acelero 5 veces
+      a340.acelerar();
+      a340.acelerar();
+      a340.acelerar();
+      a340.acelerar();
+      a340.acelerar();
+      //Solicito un estado de velocidad
+      console.log(a340.estadoVelocidad());
+      //freno 1 vez
+      a340.frenar();
+      //Solicito un estado de velocidad
+      console.log(a340.estadoVelocidad());
+
+      B787 = new Aviones();
+      B787.nombre = "Boeing-787";
+      B787.velocidad = 40;
+      B787.asientos = 240;
+      console.log(B787.saludo());
+      console.log(B787.lugares());
+      console.log(B787.estadoVelocidad());
```
Pasar a archivo javascript aparte.

cyberhavok commented on Challenge_23/JuanCarlos_Ramirez_Osorio/clases-objeto.html:
```diff
@@ -0,0 +1,84 @@
+        }
+      }
+
+      a319 = new Aviones();
```
Falta mencionar si es var, let o cons

JuanCarlosRamirez-dev committed:
Correcciones solucionadas

Creé un archivo nuevo donde agrego el script y lo llamo desde el html.
Corregí la declaración de los objetos.

JuanCarlosRamirez-dev committed:
Actividad programan
